src/foam/dao/SessionClientDAO.js

```diff
@@ -8,8 +8,8 @@ foam.CLASS({
   package: 'foam.dao',
   name: 'SessionClientDAO',
   extends: 'foam.dao.ProxyDAO',
-  javaGenerateConvenienceConstructor: false,
-  javaGenerateDefaultConstructor: false,
+  // javaGenerateConvenienceConstructor: false,
```

Comment:
You're jus stripping java functionality from this model right?

Reply:
yes, guess I should mark it as 'flags: [web]' as well. 
I initially thought the SessionClientDAO could do what the Dig client is doing but they differ.

Reply:
add flags